lib/site_search/result.rb

```diff
@@ -0,0 +1,19 @@
+module SiteSearch
+  class Result
+    attr_reader :title, :description, :link, :raw
+
+    def initialize(response_result)
+      @title = response_result[:title]
+      @description = response_result[:description]
+      @link = response_result[:link]
+      @raw = response_result[:raw]
+    end
+
+    def to_s
+      "#<SiteSearch::Result
+          @title=#{@title},
+          @link=#{@link},
+         @description=#{@description}>".squeeze
```

Comment:
is `.squeeze` the correct method here, or do you need to an argument to it? 
This will currently return
```
"#<SiteSearch:Result\n @title=<em>Budget</em> <em>planer</em>,\n @link=/en/tols/budget-planer,\n @description=You can use this <em>planer</em>.>"
```

notice how `planner` is now `planer` as it has been 'squeezed'. same for `tools`